omf/__init__.py

```diff
@@ -5,20 +5,24 @@
 from __future__ import unicode_literals
 
 from .base import Project
+
+from omf.fileio.geoh5 import GeoH5Writer
```

Comment:
**question:** does it work just with `from .fileio`? since exported by fileio/__init__.py is exporting it ?

Reply:
Yep